.github/config/linux.json

```diff
@@ -14,7 +14,8 @@
         "gin_unittests",
         "media_unittests",
         "content_shell",
-        "cobalt:gn_all"
+        "cobalt:gn_all",
+        "nplb"
     ],
```

Comment:
Is it me or this list is not in alphabetical order ? :-)

Reply:
Sorted